codebundles/k8s-postgres-query/runbook.robot

```diff
@@ -29,8 +29,11 @@ Run Postgres Query And Results to Report
     ...    workload_name=${WORKLOAD_NAME}
     ...    workload_namespace=${WORKLOAD_NAMESPACE}
     ...    workload_container=${WORKLOAD_CONTAINER}
+    ...    kubeconfig=${KUBECONFIG}
+    ...    context=${CONTEXT}
+    ...    target_service=${kubectl}
     ${rsp}=    RW.K8s.Shell
-    ...    cmd=${binary_name} exec ${workload} -- bash -c "${templated_query}" --context ${CONTEXT}
+    ...    cmd=${binary_name} exec ${workload} -- bash -c "echo '${QUERY}' > ${QUERY_FILE_PATH} && ${templated_query}" --context ${CONTEXT}
```

Comment:
Not necessary but a `Upload File To Pod` could help with readability

Reply:
Are you thinking that as a separate keyword? If so, I was combining it here to save on the amount of API calls made (e.g. avoiding one call to drop in the file and another for the execution), while also trying to keep them in the same bash execution to guarantee that they reside on the same pod. I suppose that latter concern is likely squashed by the fact that the tempalted workload uses a single pod name...